client/mass/nav.js

```diff
@@ -223,6 +226,12 @@ function setRenderers(self) {
 			titleDiv,
 			tip: new Menu({ padding: '5px' })
 		}
+		const deleteAllDiv = controlsDiv
+			.append('div')
+			.style('display', 'inline-block')
+			.style('padding-top', '4px')
+			.style('vertical-align', 'middle')
+		icon_functions['trash'](deleteAllDiv, { handler: self.deletePlots, title: 'Delete all plots' })
```

Comment:
this button is kind of dangerous. better to show a confirmation on clicking to avoid oops

Reply:
I also initially thought it's dangerous, but the undo button could easily recover the deleted plots.

Reply:
yes good point. then just modify the button `title` by adding `To revert, click Undo button`
no need to add the confirmation menu